src/rpc/daemon_handler.cpp

```diff
@@ -263,20 +263,43 @@ namespace rpc
 
   void DaemonHandler::handle(const SendRawTx::Request& req, SendRawTx::Response& res)
   {
-    auto tx_blob = cryptonote::tx_to_blob(req.tx);
+    handleTxBlob(cryptonote::tx_to_blob(req.tx), req.relay, res);
```

Comment:
I was going to criticize the `tx_to_blob` call here, but then I noticed the existing code is doing this. Something to figure out how to cleanup up in a future patch.

Reply:
It could be cleaned up by removing it and supporting only this new hex version ;)